cardano-config/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -450,6 +453,58 @@ instance ( Condense (HeaderHash blk)
   -- user defined formatting of log output
   trTransformer _ verb tr = trStructured verb tr
 
+instance DefinePrivacyAnnotation a => DefinePrivacyAnnotation (TraceLabelPeer peer a)
```

Comment:
OK, because of these generic instances?
so what did you gain? I see 78 additions and only 21 deletions in this file.

Reply:
I'll try to illustrate the difference using a commit that removes the generic instances.

Reply:
The overlaps are gone.